sarathi/til.py

```diff
@@ -385,6 +386,27 @@ def fix_urls():
                     sys.stderr.write(f"Failed getting title for `{url}`")
     write_til_json(til_json)
 
+def print_fortune(args=None):
+    import subprocess
+    import fortune
+    
+    load_dotenv()
+    
+    params = ['fortune']
+    if args != None:
+        for i in args:
+            params.append(i)
+            
+
+    FORTUNE = os.getenv(FORTUNE_DIR)
+    params.append(FORTUNE)
+
+    output = subprocess.Popen(params,stdout=subprocess.PIPE,stderr=subprocess.STDOUT)
+    stdout, stderr = output.communicate()
+    try:
+        return stdout
+    except subprocess.CalledProcessError as e:
+        return e.output
```

Comment:
Are you *sure* that `return stdout` will raise such an error, or will it get raised earlier, in the `output.communicate()` stage?

Instead of giving `e.output`, you can also think about creating your own `Exception` type.

```
class FortuneNotInstalledError(Exception):
    """This is raised whenever someone attempts to run fortune without installing it first"""

try:
    return stdout
except subprocess.CalledProcessError as error:
    raise FortuneNotInstalledError("The fortune command is not installed")
```

However, this wouldn't be good design in this case, since it's a bot we are dealing with. So what you would have to do in addition will be to *catch* this error in the bot side and then return a meaningful error to the user. However, think about this instead. When installing `sarathi`, you will need to install `fortune` also. This is something for you to think about. Don't bother about it right now.